src/main/java/Deadline.java

```diff
@@ -0,0 +1,14 @@
+public class Deadline extends Task {
+
+    protected String by;
```

Comment:
Consider changing `protected` to `private`.  In general, it is better to use `private` for instance variables for good encapsulation.